src/pages/plugins/plugins.module.scss

```diff
@@ -31,6 +30,63 @@
   font-size: 1.1rem;
 }
 
+.pluginDeprecatedCard {
+  border-radius: var(--ifm-card-border-radius);
+  box-shadow: var(--ifm-global-shadow-lw);
+  overflow: hidden;
+  background-color: #ffdfdf;
+
+  .pluginCardHeader {
+    justify-content: flex-start;
+    margin: 16px;
+    margin-bottom: 0;
+    padding: 0;
+    padding-bottom: 12px!important;
+    border-bottom: solid 1px rgba(0,0,0,.1)
+  }
+
+  h3 {
+    color: white;
+    background-color: var(--ifm-color-primary);
+    margin: .3em 0 .3em 0;
+    font-size: 0.8rem;
+    padding: 0.14rem 0.28rem;
+    margin-right: 8px;
+    font-weight: bold;
+  }
+
+  .pluginCardFlag {
+
+    .pluginCardFlagMessage {
+      width: calc(100% - 32px);
+      margin: 16px;
+      margin-top: 8px;
+    }
+
+    .deprecatedMessage {
+      color: #ff4136;
+      font-size: 0.75rem;
+      font-style: initial;
+      font-weight: 300;
+      background-color: white;
+      padding: .25rem;
+      padding-left: .5rem;
+      padding-right: .5rem;
+      float: left;
+      border-radius: .25rem;
+      border-color: rgba(0,0,0,.3);
+      border-style: solid;
+      border-width: 1px;
+      font-variant-ligatures: none;
+      font-family: "Fira Mono", "Andale Mono", "Consolas", monospace;
```

Comment:
We should just use our font already in the page.

Reply:
I deleted the font-family